packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -103,34 +127,44 @@ Search.propTypes.renderers = React.PropTypes.shape({
 });
 
 function Help({ renderers, ...props }) {
-	const Components = getRenderers(renderers);
+	const global = {
+		bsStyle: 'link',
+		icon: 'talend-question-circle',
+		label: 'Help',
+		tooltipPlacement: 'bottom',
+		...props,
```

Comment:
Just to be sure, do we want to make the props possibly override one of the previous values, or are those more important ?